specification/applicationinsights/resource-manager/Microsoft.Insights/stable/2015-05-01/myworkbooks_API.json

```diff
@@ -0,0 +1,554 @@
+          "type": "string",
+          "description": "Azure resource Id",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "name": {
+          "type": "string",
+          "description": "Azure resource name",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "type": {
+          "type": "string",
+          "description": "Azure resource type",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "location": {
+          "type": "string",
+          "description": "Resource location",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "tags": {
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "x-ms-azure-resource": true,
+      "description": "An azure resource object"
+    },
+    "MyWorkbooksListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "readOnly": true,
+          "items": {
+            "$ref": "#/definitions/MyWorkbook"
+          },
+          "description": "An array of private workbooks."
+        }
+      },
+      "description": "Workbook list result."
+    },
+    "MyWorkbook": {
+      "description": "An Application Insights private workbook definition.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/MyWorkbookResource"
+        }
+      ],
+      "properties": {
+        "kind": {
+          "type": "string",
+          "description": "The kind of workbook. Choices are user and shared.",
+          "enum": [
+            "user",
+            "shared"
+          ],
+          "x-ms-enum": {
+            "name": "SharedTypeKind",
+            "modelAsString": true
+          }
+        },
+        "properties": {
+          "x-ms-client-flatten": true,
+          "description": "Metadata describing a workbook for an Azure resource.",
+          "$ref": "#/definitions/MyWorkbookProperties"
+        }
+      }
+    },
+    "MyWorkbookProperties": {
+      "description": "Properties that contain a private workbook.",
+      "required": [
+        "displayName",
+        "kind",
+        "category",
+        "serializedData",
+        "userId"
+      ],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "The user-defined name of the private workbook."
+        },
+        "serializedData": {
+          "type": "string",
+          "description": "Configuration of this particular private workbook. Configuration data is a string containing valid JSON"
+        },
+        "version": {
+          "type": "string",
+          "description": "This instance's version of the data model. This can change as new features are added that can be marked private workbook."
+        },
+        "kind": {
```

Comment:
Why do we have two `kind`s?

Reply:
shouldn't be here.  removing.  the top level one is correct.  i think a lot of this never got fixed in the previous review of workbooks (which i wasn't directly involved in last time). 

i've already removed some of the other ones, like "workbookId" was here, but "sourceId" wasn't, and those still (incorrectly) appear in the workbooks_api.json.  I'd like to fix them but those would DEFINATELY show up as a breaking change, right?